src/aap_eda/api/serializers/activation.py

```diff
@@ -130,25 +147,26 @@ class ActivationCreateSerializer(serializers.ModelSerializer):
     )
     user = serializers.HiddenField(default=serializers.CurrentUserDefault())
 
+    awx_token_id = serializers.IntegerField(
+        allow_null=True,
+        validators=[validators.check_if_awx_token_exists],
+        required=False,
```

Comment:
@Alex-Izquierdo if the required is `false` and it can be nullable, do we still need validators?